BranchMetrics/android-branch-deep-linking-attribution
# [SDK-1998] Replaced the share sheet's userCompletedAction call with v2/event

## Reference
SDK-1998 --[Android] Remove the userCompletedAction call from showShareSheet

## Description
After a share sheet option was chosen, `userCompletedAction` was called to send a share event. Since `userCompletedAction` is deprecated, the event has been updated to a v2/event.

## Testing Instructions
Use share sheet methods and observe the new v2/event with the appropriate fields.

## Risk Assessment [`LOW`]




## Reviewer Checklist (To be checked off by the reviewer only)

- Correctness & Style

cc @BranchMetrics/saas-sdk-devs for visibility.
